published_topics:
  ledstrip_state: /kinect/ledstrip/state

subscribed_topics:
  ledstrip_cmd: /kinect/ledstrip/state_cmd
  brightness: /kinect/ledstrip/brightness
  state_client: /robot/state/clients

led_gpio_pin: 22
pub_rate: 1
